internal_ws/ykcompile/src/lib.rs

```diff
@@ -57,9 +57,17 @@ lazy_static! {
     static ref ICTX_REG: u8 = RDI.code();
 
     static ref TEMP_LOC: Location = Location::Reg(*TEMP_REG);
+
+    /// The size of a memory address in bytes.
     static ref PTR_SIZE: u64 = u64::try_from(mem::size_of::<usize>()).unwrap();
 }
 
+/// The size of a quad-word register (e.g. RAX) in bytes.
+const QWORD_REG_SIZE: usize = 8;
+
+/// Number of bytes the stack must be aligned by, immediately after a call via the Sys-V ABI.
```

Comment:
This is incorrect. It should be: `Bytes the stack must be aligned to, immediately before a call via the Sys-V ABI.`

Reply:
Fixed in 8681961